awesomplete.js

```diff
@@ -304,6 +304,8 @@ function Suggestion(data) {
 	  ? { label: data[0], value: data[1] }
 	  : typeof data === "object" && "label" in data && "value" in data ? data : { label: data, value: data };
 
+	for (var attr in data) { this[attr] = data[attr]; }
```

Comment:
- [x] Please follow the style of the rest of the code, with blocks being on separate lines even if they only have one line.
- [x] Please rename attr to prop or (better) property. These are not attributes, they are properties. Attributes can be confused with HTML attributes.

```suggestion
	if (typeof data === "object") {
		for (var property in data) {
			this[property] = data[property];
		}
	}
```